packages/cli/src/config.ts

```diff
@@ -0,0 +1 @@
+export { defineConfig, type PrismaConfig, type PrismaConfigInternal } from '@prisma/config'
```

Comment:
`PrismaConfigInternal` must be exported, otherwise

```typescript
import { defineConfig } from 'prisma/config'
```

will result in a

```
The inferred type of `default` cannot be named without a reference to `...`. This is likely not portable. A type annotation is necessary.
```

error.